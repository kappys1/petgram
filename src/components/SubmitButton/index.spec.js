// Category.test.js
/* global describe, it, expect */
import React from 'react'
// import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react'
import { SubmitButton } from '.'

describe('SubmitButton', () => {
  it('snapshot', () => {
    const { asFragment } = render(<SubmitButton><span>test</span></SubmitButton>)
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render spinner', () => {
    render(<SubmitButton isLoading>test</SubmitButton>)
    expect(screen.getByTitle('loading')).toBeInTheDocument()
  })
})
