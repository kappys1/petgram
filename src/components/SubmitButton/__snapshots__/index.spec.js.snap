// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton snapshot 1`] = `
<DocumentFragment>
  <button
    class="sc-bdfBQB emfePF"
  >
    <span>
      test
    </span>
  </button>
</DocumentFragment>
`;
